x-pack/plugins/infra/server/routes/metrics_explorer/lib/convert_metric_to_metrics_api_metric.test.ts

```diff
@@ -34,32 +34,13 @@ describe('convertMetricToMetricsAPIMetric(metric, index)', () => {
   runTestForBasic('cardinality');
 
   runTest(
-    { aggregation: 'rate', field: 'system.network.in.bytes' },
+    { aggregation: 'avg', field: 'host.network.ingress.bytes' },
     {
       id: 'metric_1',
       aggregations: {
-        metric_1_max: {
```

Comment:
I think we should revert this, it's really testing to see if it converts the Metrics Explorer's `rate` aggregation to the correct aggregations used in Elasticsearch; The test isn't specific to `system.network.in.bytes`. A user could use any "counter" field with the `rate` aggregation in the Metrics Explorer.